rollup.config.js

```diff
@@ -29,13 +28,12 @@ module.exports = [
 			resolve(),
 			commonjs(),
 			babel(),
-			stylesheet({
-				extract: true
-			}),
 			optional({
 				include: ['moment']
 			}),
-			cleanup(),
+			cleanup({
+				sourcemap: true
```

Comment:
should this be added below in the other place we're calling `cleanup` as well?

Reply:
Not sure if any one needs it, but sure. This one is needed for tests.